### ISSUE TITLE
Different version min vs src

### ISSUE THREAD

**Issue by totpero** [gasparesganga/jquery-popup-window#9]
Hi,

In min file is specified [1.0.4 version](https://github.com/gasparesganga/jquery-popup-window/blob/master/src/popupwindow.min.js#L4) and in src is [1.0.3 version](https://github.com/gasparesganga/jquery-popup-window/blob/master/src/popupwindow.js#L4).
The same is on css files not only js files.
Is any differences between this tow versions?

**Comment by totpero** [comment]
Yes, is differences. I include the src version into my project and I get errors.
Can you publish the 1.04 source not only the min version?

**Comment by gasparesganga** [comment]
Hi. Unfortunately I haven't been updating this project since 2019.
I had a new version in the pipeline but due to work schedule and personal issues during this time, I kind of set it aside for the moment, I'm sorry.

A quick look at the code suggests me that release [v1.1.0](https://github.com/gasparesganga/jquery-popup-window/releases/tag/v1.1.0) should include that change purposed by pull request #7 even though the version numbers in the headers are messed up (code for that release was added by an external contributor).

If you don't need those features included in v1.1.0 simply use [v1.0.3](https://github.com/gasparesganga/jquery-popup-window/releases/tag/v1.0.3) and you should have no bugs whatsoever. I have been using it in my personal projects for years without any issue actually.
